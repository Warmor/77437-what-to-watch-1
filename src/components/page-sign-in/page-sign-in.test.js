import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {shallowToJson} from 'enzyme-to-json';
import PageSignIn from "~/components/page-sign-in/page-sign-in";

Enzyme.configure({adapter: new Adapter()});

describe(`<PageSignIn/>`, () => {
  it(`PageSignIn renders correctly`, () => {
    const tree = shallow(<PageSignIn/>);
    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});
