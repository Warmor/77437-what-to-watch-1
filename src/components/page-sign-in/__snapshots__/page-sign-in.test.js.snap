// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageSignIn/> PageSignIn renders correctly 1`] = `
<Fragment>
  <div
    className="user-page"
  >
    <ConnectFunction
      headerTitle="Sign in"
    />
    <ConnectFunction />
    <Footer />
  </div>
</Fragment>
`;
